### PR TITLE
Draw svg title

### PR DESCRIPTION
**Pull request by hyanwong**
I've always wanted to be able to add a title to our SVG plots. Now that we are outputting them e.g. from sc2ts, I think this is vital to keep track of what we are plotting:

Screenshot 2024-10-08 at 23 04 10

### CODE REVIEW

**Review by benjeffery — APPROVED** [review]
LGTM!

**Review by jeromekelleher — APPROVED** [review]

File: python/tskit/intervals.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # MIT License
 #
-# Copyright (c) 2023 Tskit Developers
+# Copyright (c) 2024 Tskit Developers
```
Comment (jeromekelleher) [inline python/tskit/intervals.py:3]: Not sure why this file got touched?

**Review by jeromekelleher — COMMENTED** [review]
Looks like you need to drop intervals.py from the PR?

File: python/tskit/intervals.py (hunk 1 above)
Comment (hyanwong) [reply python/tskit/intervals.py:3]: It didn't, but one of the CI checks for some reason was complaining about in in a previous PR. Happy to remove

Comment (hyanwong) [reply python/tskit/intervals.py]: I just removed the commit that did the year updates for extra untouched files that were previously complained about.

**Pull request merged**